packages/tasit-contracts/src/scripts/populateDecentralandContracts.js

```diff
@@ -78,263 +213,184 @@ const createEstate = async (estateContract, landContract, estate, wallet) => {
 
-(async () => {
-  const config = require(`../config/${network}.js`);
-
-  // https://stats.goerli.net/
-  if (network === "goerli")
-    gasParams = {
-      gasLimit: 8e6,
-      gasPrice: 1e10,
-    };
-  else if (network === "development") {
-    network = "local";
-  }
-  const {
-    LANDProxy,
-    EstateRegistry,
-    Marketplace,
-    GnosisSafe: GnosisSafeInfo,
-    MANAToken,
-  } = TasitContracts[network];
-  const { address: MANA_ADDRESS } = MANAToken;
-  const { address: LAND_PROXY_ADDRESS } = LANDProxy;
-  const { address: ESTATE_ADDRESS } = EstateRegistry;
-  const { address: MARKETPLACE_ADDRESS } = Marketplace;
-  const { address: GNOSIS_SAFE_ADDRESS } = GnosisSafeInfo;
-
-  ConfigLoader.setConfig(config);
-
-  const [minterWallet, sellerWallet] = accounts;
-  const { address: sellerAddress } = sellerWallet;
-
-  const manaContract = new Mana(MANA_ADDRESS);
-  const landContract = new Land(LAND_PROXY_ADDRESS);
-  const estateContract = new Estate(ESTATE_ADDRESS);
-  const marketplaceContract = new Decentraland(MARKETPLACE_ADDRESS);
-  const gnosisSafeContract = new GnosisSafe(GNOSIS_SAFE_ADDRESS);
-
-  // Fund Gnosis Safe wallet with Mana tokens and ethers
-  const provider = ProviderFactory.getProvider();
-  await etherFaucet(provider, minterWallet, GNOSIS_SAFE_ADDRESS, ONE);
-  await erc20Faucet(
-    manaContract,
-    minterWallet,
-    GNOSIS_SAFE_ADDRESS,
-    ONE_THOUSAND
+const findParcel = (parcel, listOfParcels) => {
+  return listOfParcels.find(p => parcelsAreEqual(p, parcel));
+};
+
+const estateContainsParcelFromList = (estate, listOfParcels) => {
+  const { parcels: estateParcels } = estate;
+
+  for (let parcel of estateParcels)
+    if (findParcel(parcel, listOfParcels)) return true;
+
+  return false;
+};
+
+const getEstatesFromAPI = async () => {
+  const res = await fetch(
+    "https://api.decentraland.org/v1/estates?status=open&limit=100"
+  );
+  const json = await res.json();
+  const { data: jsonData } = json;
+  const { estates: estatesFromAPI } = jsonData;
+
+  const allEstates = estatesFromAPI.map(estate => {
+    const { data } = estate;
+    const { name: metadata, parcels } = data;
+    return { metadata, parcels };
+  });
+
+  // Keep estates with small number of parcels to avoid out-of-gas on creation
+  const estates = allEstates.filter(e => e.parcels.length < 10);
+  return estates;
+};
+
+const getParcelsFromAPI = async () => {
+  const res = await fetch(
+    "https://api.decentraland.org/v1/parcels?status=open&limit=100"
   );
+  const json = await res.json();
+  const { data: jsonData } = json;
+  const { parcels: parcelsFromAPI } = jsonData;
+
+  const parcels = parcelsFromAPI.map(parcel => {
+    const { x, y, data } = parcel;
+    let { name: metadata } = data;
+    if (!metadata) metadata = "";
+    return { x, y, metadata };
+  });
 
-  const allParcels = [
-    // Estate: all road adjacent parcels
-    { x: -30, y: -105, metadata: `` },
```

Comment:
If we make a unit test suite for this script, this data could be used to mock an API response

Reply:
Yes, I agree. Worth keep that? It's no big deal since I've got these data in a few minutes looking for the Decentraland marketplace.

Reply:
Nah, we don't need to keep it. Good to know it's easy to get back (which makes sense). I'll resolve this.